src/axom/sidre/core/Group.hpp

```diff
@@ -1358,6 +1405,21 @@ class Group
    */
   bool createNativeLayout(Node& n, const Attribute* attr = nullptr) const;
 
+  /*!
+   * \brief Deep-copy Group's native layout to given Conduit node.
+   *
+   * This is similar to createNativeLayout, except for the leaves being
+   * deep-copied.
+   *
+   * The destination's allocator id should be preserved and used for
+   * array allocations.  However, I'm still checking on this.
```

Comment:
This work-in-progress comment should be resolved prior to merge.

Reply:
Good catch.  That is obviously a note to myself to check it.  I did check but forgot to remove the note.